Documentation/quickstart/index.rst

```diff
@@ -1,16 +1,18 @@
 .. todo::
-  This should be a brief introduction to how NuttX works and how to get it running. A widely used board could be targeted to create a brief demonstration on how to clone, configure, build, flash and run NuttX.
-  For now, the content that is here presented is what remains of the old Porting Guide which is not already in another section of this documentation.
 
 ==========
 Quickstart
 ==========
 
-The present Quickstart guide is a work in progress and only contains some basic information. In the meantime you can look at NuttX main `README <https://github.com/apache/incubator-nuttx/blob/master/README.md>`_ file which contains some information that will help you get started.
```

Comment:
It is awkward to have a section be empty and only have the child documents be listed in the TOC. I think it is a good practice to always have something to say about the section. Could you describe the purpose of this "getting started" section, mention that you can start with quickstart and then the other ones go over into more detail? Don't be afraid of being repetitive or verbose, I feel it makes it more welcoming to newcomers.

Reply:
@v01d Will do.